src/Command/Analyze.php

```diff
@@ -43,7 +43,7 @@ protected function configure(): void
             ->setHelp('');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output): void
+    protected function execute(InputInterface $input, OutputInterface $output): ?int
```

Comment:
Why nullable? The only returns I see now are ints.

Reply:
The base function defines `@return null|int null or 0 if everything went fine, or an error code`. I wanted to respect this

Reply:
fixed